docs/details/main-components/color.rst

```diff
@@ -135,13 +139,15 @@ Some special purpose defines are also introduced:
    completely transparent
 -  :cpp:enumerator:`LV_OPA_10` Value: 25, means the color covers only a little
 -  ``LV_OPA_20 ... OPA_80`` follow logically
--  :cpp:enumerator:`LV_OPA_90` Value: 229, means the color near completely covers
+-  :cpp:enumerator:`LV_OPA_90` Value: 229, means the color nearly completely covers
 -  :cpp:enumerator:`LV_OPA_COVER` Value: 255, means the color completely covers (full
    opacity)
 
 You can also use the ``LV_OPA_*`` defines in :cpp:func:`lv_color_mix` as a
 mixing *ratio*.
 
+
+
```

Comment:
```suggestion
```

Reply:
Hi, Gábor!  ( @kisvegabor )

I have been having exactly 3 blank lines above chapters (\*\*\*\*\*\*) (except if one appears immediately below the title) because they are "major" sections, and it helps to visually differentiate the "major" sections from the "minor" sections in the document.  I found that 2 blank lines is "good", but 3 makes it easier to see (at a glance) where they are.   I think I have been about 98% consistent about having exactly 3 blank lines, just for readability.  (Normally I have been having 2 blank lines [in compliance with `/docs/README.md`] between sub-sections below chapters, but sometimes, especially if there are not a lot of changes in the text, I skipped doing this simply to reduce the number of changes, when the sub-sections are easy to see where they are.  `color.rst` is an example of that.)

Let me know if you have any strong objections to that practice.

Reply:
If it's a consistent style, than it's good :slightly_smiling_face:

Reply:
I thought so as well.  :-)